website/content/docs/templates/hcl_templates/blocks/build/provisioner.mdx

```diff
@@ -135,8 +135,8 @@ build {
 ```
 
 As you can see, the `override` key is used. The value of this key is another
-HCL attribute map where the key is the name of a [source
-definition](/docs/templates/hcl_templates/blocks/source). The value of this is in turn
+HCL attribute map where the key is the name of a [builder
```

Comment:
```suggestion
HCL attribute map where the key is the name of a [source
```
To resolve with updates from `main`

Reply:
to not overwrite https://github.com/hashicorp/packer/pull/12220, must have been messed up in the merge

Reply:
To not overwrite https://github.com/hashicorp/packer/commit/ba546eb8cbeed152a4c1211b7022f32e78cdefe3, must been messed up in the merge

Reply:
Good catch.